lib/lotus/utils/inflector.rb

```diff
@@ -395,90 +259,74 @@ def apply(string)
-        'sassafrases'  => 'sassafras',
-        'canvases'     => 'canvas',
-        'ibises'       => 'ibis',
-        'trellises'    => 'trellis',
-        'horses'       => 'horse',
         # fallback
-        'hives' => 'hive',
-        # ices => ex
-        "codices"    => "codex",
-        "murices"    => "murex",
-        "silices"    => "silex",
-        "apices"     => "apex",
-        "latices"    => "latex",
-        "vertices"   => "vertex",
-        "cortices"   => "cortex",
-        "pontifices" => "pontifex",
-        "vortices"   => "vortex",
-        "indices"    => "index",
-        "simplices"  => "simplex",
-        # ices => ix
-        "radices"    => "radix",
-        "helices"    => "helix",
-        "appendices" => "appendix",
-        # es => is
-        "axes"        => "axis",
-        "analyses"    => "analysis",
-        "bases"       => "basis",
-        "crises"      => "crisis",
-        "diagnoses"   => "diagnosis",
-        "ellipses"    => "ellipsis",
-        "hypotheses"  => "hypothesis",
-        "oases"       => "oasis",
-        "paralyses"   => "paralysis",
-        "parentheses" => "parenthesis",
-        "syntheses"   => "synthesis",
-        "synopses"    => "synopsis",
-        "theses"      => "thesis",
+        'hives'       => 'hive',
+        'horses'      => 'horse',
       })
 
+      # Block for custom inflection rules.
+      #
+      # @param [blk] custom inflections
+      #
+      # @since x.x.x
+      #
+      # @see Lotus::Utils::Inflector.exception
+      # @see Lotus::Utils::Inflector.uncountable
+      #
+      # @example
+      #   require 'lotus/utils/inflector'
+      #
+      #   Lotus::Utils::Inflector.inflections do
+      #     exception   'analysis', 'analyses'
+      #     exception   'alga',     'algae'
+      #     uncountable 'music', 'butter'
+      #   end
+      def self.inflections(&blk)
+        class_eval(&blk)
+      end
+
+      # Add a custom inflection exception
+      #
+      # @param [String] singular form
+      # @param [String] plural form
+      #
+      # @since x.x.x
+      #
+      # @see Lotus::Utils::Inflector.inflections
+      # @see Lotus::Utils::Inflector.uncountable
+      #
+      # @example
+      #   require 'lotus/utils/inflector'
+      #
+      #   Lotus::Utils::Inflector.inflections do
+      #     exception 'alga', 'algae'
+      #   end
+      def self.exception(singular, plural)
+        singulars.add(plural, singular)
+        plurals.add(singular, plural)
+      end
+
+      # Add an uncountable word
+      #
+      # @param [String] word
+      #
+      # @since x.x.x
+      #
+      # @see Lotus::Utils::Inflector.inflections
+      # @see Lotus::Utils::Inflector.exception
+      #
+      # @example
+      #   require 'lotus/utils/inflector'
+      #
+      #   Lotus::Utils::Inflector.inflections do
+      #     uncountable 'music', 'art'
+      #   end
+      def self.uncountable(*words)
+        Array(words).each do |word|
```

Comment:
@AlfonsoUceda When you use the splat operator, you can both call with:

```
uncountable 'money'
uncountable 'music', 'art'
```

That call to `Array()` ensures that the iterating object is an array, regardless of the argument value.